From 4346239a5d9d34761885009ba65a4e3231a45182 Mon Sep 17 00:00:00 2001
From: Tiago Vignatti <tiago.vignatti@intel.com>
Date: Thu, 22 Nov 2012 20:22:56 -0200
Subject: [PATCH 05/39] configure: Track updated version of libxtrans

XWayland fails to initialize in some systems complaining about realloc
problems on libxtrans (when ListenOnOpenFD() is called). It got fixed in
libxtrans version 1.2.7, more specifically:

    commit 6086f6c1d0e0a1c9e590879acb2319dea0eb6e96
    Author: Robert Bragg <robert@linux.intel.com>
    Date:   Mon Dec 12 00:30:43 2011 +0000

Reported-by: nerdopolis, Prf_Jakob, spitzak among others at #wayland channel
Signed-off-by: Tiago Vignatti <tiago.vignatti@intel.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6925df8..9fb014f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -806,7 +806,7 @@ FIXESPROTO="fixesproto >= 5.0"
 DAMAGEPROTO="damageproto >= 1.1"
 XCMISCPROTO="xcmiscproto >= 1.2.0"
 BIGREQSPROTO="bigreqsproto >= 1.1.0"
-XTRANS="xtrans >= 1.2.2"
+XTRANS="xtrans >= 1.2.7"
 PRESENTPROTO="presentproto >= 1.0"
 
 dnl List of libraries that require a specific version
-- 
1.8.3.1

